app/controllers/PenaltiesFrontendController.scala

```diff
@@ -108,11 +108,9 @@ class PenaltiesFrontendController @Inject()(auditService: AuditService,
   }
 
   private def returnResponse(penaltyDetails: GetPenaltyDetails, enrolmentKey: String, arn: Option[String] = None)(implicit request: Request[_]): Result = {
-    if (penaltyDetails.lateSubmissionPenalty.map(_.summary.activePenaltyPoints).getOrElse(0) > 0) {
-      val auditModel = UserHasPenaltyAuditModel(penaltyDetails, RegimeHelper.getIdentifierFromEnrolmentKey(enrolmentKey),
-        RegimeHelper.getIdentifierTypeFromEnrolmentKey(enrolmentKey), arn)
-      auditService.audit(auditModel)
-    }
+    val auditModel = UserHasPenaltyAuditModel(penaltyDetails, RegimeHelper.getIdentifierFromEnrolmentKey(enrolmentKey),
+      RegimeHelper.getIdentifierTypeFromEnrolmentKey(enrolmentKey), arn)
+    auditService.audit(auditModel)
```

Comment:
I think we still need the if statement but we just need to include a check for LPPs like below. This will avoid audit events being sent for users who have no LSPs and no LPPs
```suggestion
    val hasLSP = penaltyDetails.lateSubmissionPenalty.map(_.summary.activePenaltyPoints).getOrElse(0) > 0
    val hasLPP = penaltyDetails.latePaymentPenalty.flatMap(_.details.map(_.length)).getOrElse(0) > 0
    
    if (hasLSP || hasLPP) {
      val auditModel = UserHasPenaltyAuditModel(penaltyDetails, RegimeHelper.getIdentifierFromEnrolmentKey(enrolmentKey),
        RegimeHelper.getIdentifierTypeFromEnrolmentKey(enrolmentKey), arn)
      auditService.audit(auditModel)
    }
    Ok(Json.toJson(penaltyDetails))
  }
```